ckanext/canada/view.py

```diff
@@ -92,12 +108,39 @@ def post(self, package_type, id):
         return response
 
 
+class CanadaUserRegisterView(UserRegisterView):
+    def post(self):
+        params = parse_params(request.form)
+        email=params.get('email', '')
+        fullname=params.get('fullname', '')
+        username=params.get('name', '')
+        phoneno=params.get('phoneno', '')
+        dept=params.get('department', '')
+        response = super(CanadaUserRegisterView, self).post()
+        if hasattr(response, 'status_code'):
+            if response.status_code == 200 or response.status_code == 302:
+                # redirected after successful user create
+                import ckan.lib.mailer
+                # checks if there is a custom function "notify_ckan_user_create" in the mailer (added by ckanext-gcnotify)
+                getattr(
+                  ckan.lib.mailer,
```

Comment:
let's use 4-space indents throughout python code in the repo

Reply:
@wardi oops, just copied that code over from the Controller. Is fixed now.

Reply:
yep, might as well clean it up now.